res/css/views/settings/tabs/room/_SecurityRoomSettingsTab.scss

```diff
@@ -16,24 +16,12 @@ limitations under the License.
 
 .mx_SecurityRoomSettingsTab {
     .mx_SettingsTab_showAdvanced {
-        padding: 0;
-        margin-bottom: 16px;
+        margin-bottom: $spacing-16;
     }
-}
-
-.mx_SecurityRoomSettingsTab_warning {
-    display: block;
 
-    img {
-        vertical-align: middle;
-        margin-right: 5px;
-        margin-left: 3px;
-        margin-bottom: 5px;
+    .mx_SecurityRoomSettingsTab_warning {
+        display: flex;
+        align-items: center;
+        column-gap: $spacing-4;
     }
 }
-
-.mx_SecurityRoomSettingsTab_encryptionSection {
```

Comment:
This was added with https://github.com/matrix-org/matrix-react-sdk/pull/2523/files#diff-32beddecb1bd760a65b15adab68b83321fff88243061955848e3bca9c4554048R32, and this selector is no longer used.